ctapipe/io/dl1eventsource.py

```diff
@@ -153,21 +154,27 @@ def datalevels(self):
         elif images:
             return (DataLevel.DL1_IMAGES,)
 
-    @property
+    @lazyproperty
     def obs_ids(self):
         return list(np.unique(self.file_.root.dl1.event.subarray.trigger.col("obs_id")))
 
     @property
-    def mc_headers(self):
-        return self._mc_headers
+    def simulation_config(self):
+        """
+        Returns the simulation config.
+        In case of a merged file, this will be a list of simulation configs.
+        """
+        if len(self._simulation_configs) == 1:
```

Comment:
I would not special case this. Just always return a list or better maybe `dict(obs_id -> config)`?

Reply:
I wanted this to behave the same way as the SimtelEventSource for unmerged files.
But we can change it over there as well of course

Reply:
I think simtel files can also contain multiple corsika runs...

Reply:
Then it becomes an api change for sure though as other EventSources change as well

Reply:
Ok, than I'd say leave it as is.